megamek/src/megamek/server/GameManagerSaveHelper.java

```diff
@@ -95,8 +108,10 @@ public void sendSaveGame(int connId, String fileName, String localPath) {
                 finalFileName = fileName + MMConstants.SAVE_FILE_GZ_EXT;
             }
         }
-        localPath = localPath.replaceAll("\\|", " ");
+
+        localPath = localPath.replace("|", " ");
```

Comment:
Was the escaping of the "|" actually unnecessary here?

Reply:
`replaceAll` defaults to a regex pattern and create unneeded overhead if it's just a string.
`replace` defaults to string and only moves to regex when needed.